test/EventStore.Client.PersistentSubscriptions.Tests/SubscriptionToAll/list_persistent_subscriptions.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EventStore.Client.SubscriptionToAll {
+	public class list : IClassFixture<list.Fixture> {
+		private readonly Fixture _fixture;
+		private const int AllStreamSubscriptionCount = 3;
+		private const int StreamSubscriptionCount = 4;
+		private const int TotalSubscriptionCount = AllStreamSubscriptionCount + StreamSubscriptionCount;
+		private const string GroupName = nameof(list);
+		private const string StreamName = nameof(list);
+
+		public list(Fixture fixture) {
+			_fixture = fixture;
+		}
+
+		[Fact]
+		public async Task returns_subscriptions_to_all_stream() {
+			var result = await _fixture.Client.ListForAllAsync(TestCredentials.Root).ToListAsync();
```

Comment:
**SUGGESTION:** I think that if there were HTTP fallbacks added, then the regression tests below would not fail.

Reply:
Test was failing because of changes from other PR, fallback added.